### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mlescaudron**
Fixer.io need an api key > https://fixer.io/documentation

### CODE REVIEW

**Comment by alxpez** [comment]
> \* Currently, the only accepted inputs for the `Auth` field are as follows:
> 
> * `OAuth` - _the API supports OAuth_
> * `apiKey` - _the API uses a private key string/token for authentication - try and use the correct parameter_
> * `X-Mashape-Key` - _the name of the header which may need to be sent_
> * `No` - _the API requires no authentication to run_

File: README.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | 1Forge | Forex currency market data | `apiKey` | Yes | Unknown | [Go!](https://1forge.com/forex-data-api/api-documentation) |
 | Currencylayer | Exchange rates and currency conversion | `apiKey` | Yes | Unknown | [Go!](https://currencylayer.com/documentation) |
 | Czech National Bank | A collection of exchange rates | No | Yes | Unknown | [Go!](https://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.xml) |
-| Fixer.io | Exchange rates and currency conversion | No | Yes | Unknown | [Go!](http://fixer.io) |
```
Comment (alxpez) [inline README.md]: It does not follow the repo guidelines

**Comment by davemachado** [comment]
Hi @mlescaudron , thanks for the PR. Please update the Auth field to `apiKey`.

**Comment by davemachado** [comment]
Bump

**Comment by davemachado** [comment]
Fixed in 0274029d96e966e2766b78b35928b9273f76a592

**Pull request closed without merging**